data/filedefs/filetypes.javascript

```diff
@@ -3,8 +3,8 @@
 
 [keywords]
 # all items must be in one line
-primary=break case catch class const continue default delete do each else extends false finally for function get if in Infinity instanceof let NaN new null return set static super switch this throw true try typeof undefined var void while with yield prototype
-secondary=Array Boolean Date Function Math Number Object String RegExp EvalError Error RangeError ReferenceError SyntaxError TypeError URIError constructor prototype decodeURI decodeURIComponent encodeURI encodeURIComponent eval isFinite isNaN parseFloat parseInt
+primary=break case catch class const continue debugger default delete do else enum export extends false finally for function get if import in Infinity instanceof let NaN new null return set static super switch this throw true try typeof undefined var void while with yield prototype async await
+secondary=Array Boolean Date Function Math Number Object String RegExp EvalError Error RangeError ReferenceError SyntaxError TypeError URIError constructor prototype decodeURI decodeURIComponent encodeURI encodeURIComponent eval isFinite isNaN parseFloat parseInt alert
```

Comment:
Adding `alert` is somewhat questionable, especially as I'm not really sure it's available outside browsers.  I don't know with Node, but GJS doesn't have it.
And according to https://developer.mozilla.org/en-US/docs/Web/API/Window/alert it seems to be a DOM thing, not a plain JS one.